resources/home/dnanexus/eggd_artemis.py

```
@@ -2,178 +1,29 @@
 import dxpy
 import datetime
 import logging
-from collections import defaultdict
-
-from glob import glob
 import pip
 
 pip.main(["install", "--no-index", "--no-deps", *glob("packages/*")])
 
-
 from mergedeep import merge
```

Ruff (0.8.0)

Module level import not at top of file

(E402)